### PR TITLE
docs: add template files for community

### PR DESCRIPTION
**Pull request by An-DJ**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE /BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report    
+
+about: If you would like to report an issue to seata-ctl, please use this template.
```
Comment (slievrly) [inline .github/ISSUE_TEMPLATE /BUG_REPORT.md:4]: this project is not seata-ctl.

File: .github/ISSUE_TEMPLATE /BUG_REPORT.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Minimal yet complete reproducer code (or URL to code):
+
+
+
+### Ⅴ. Anything else we need to know?
+
+
+### Ⅵ. Environment:
+
+- JDK version(e.g. `java -version`):
```
Comment (slievrly) [inline .github/ISSUE_TEMPLATE /BUG_REPORT.md:41]: Suggest changing it, it's not a java project.

File: .github/ISSUE_TEMPLATE /BUG_REPORT.md (hunk 1 above)
Comment (An-DJ) [reply .github/ISSUE_TEMPLATE /BUG_REPORT.md]: Fixed.

File: .github/ISSUE_TEMPLATE /BUG_REPORT.md (hunk 2 above)
Comment (An-DJ) [reply .github/ISSUE_TEMPLATE /BUG_REPORT.md]: Replaced with 'Go version'

File: .github/ISSUE_TEMPLATE /BUG_REPORT.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+name: Bug Report    
+
+about: If you would like to report an issue to Seata, please use this template.
+
+---
+<!-- Please do not use this issue template to report security vulnerabilities but refer to our [security policy](https://github.com/seata/seata/security/policy). -->
+
+- [ ] I have searched the [issues](https://github.com/seata/seata-ctl/issues) of this repository and believe that this is not a duplicate.
```
Comment (slievrly) [inline .github/ISSUE_TEMPLATE /BUG_REPORT.md:9]: rename as apache/incubator-seata-ctl

Comment (slievrly) [reply .github/ISSUE_TEMPLATE /BUG_REPORT.md:9]: Please check all rename cases.

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**